pages/common/picom-trans.md

```diff
@@ -0,0 +1,20 @@
+# picom-trans
+
+> Set window opacity.
+> More information: <https://github.com/yshui/picom>.
+
+- Set the currently focused window opacity to a certain percentage:
```

Comment:
```suggestion
- Set the current window opacity to a specific percentage:
```

AFAIK we prefer `a` instead of `the`, and `specific` instead of `certain`. Pages should be standardized. ;)

Reply:
I agree with `specific`, but what's wrong with `currently focused`. It's more precise than `current`.

Reply:
1) What about [Simon's](https://github.com/tldr-pages/tldr/pull/8244#discussion_r928295348) remark about "focussed"? I think we should incorporate this.
2) Okay, but I'm already using `a` (at least in the descriptions below). I think using an indefinite article twice doesn't make much sense here: I'm talking a about **a** window, so it's **the** opacity of this particular window (definite in this context).
3) Okay, I will change `certain` to `specific`. (Wanted to avoid using `specific` twice in the second description)